src/components/Chips/Chip.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+const propTypes = {
+  tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+  className: PropTypes.string,
+  simple: PropTypes.bool,
+  large: PropTypes.bool,
+  disabled: PropTypes.bool
+}
+
+const defaultProps = {
+  tag: 'div',
+  simple: false,
+  large: false,
+  disabled: false
+}
+
+const Chip = props => {
+  const { className, tag: Tag, simple, large, disabled, ...attributes } = props
+  const wrapperClasses = classNames(
+    className,
+    simple ? 'chip-simple' : false,
+    large ? 'chip-lg' : false,
+    disabled ? 'chip-disabled' : false,
```

Comment:
could you use the object version for this? Having this multiple `false` makes it harder to read.

Something like:
```js
const wrapperClasses = classNames(
  'chip',
  className,
  {
    'chip-simple': simple,
    'chip-lg': large,
    'chip-disabled': disabled
  }
);
```

Reply:
Why then `wrapperClasses` as variable name? Maybe `classes` here is enough.

Reply:
Done